shell/platform/fuchsia/flutter/accessibility_bridge.cc

```diff
@@ -115,6 +115,36 @@ fuchsia::accessibility::semantics::States AccessibilityBridge::GetNodeStates(
   return states;
 }
 
+std::vector<fuchsia::accessibility::semantics::Action>
+AccessibilityBridge::GetNodeActions(const flutter::SemanticsNode& node,
+                                    size_t* additional_size) const {
+  std::vector<fuchsia::accessibility::semantics::Action> node_actions;
+
+  if (node.HasAction(flutter::SemanticsAction::kTap)) {
+    node_actions.push_back(fuchsia::accessibility::semantics::Action::DEFAULT);
```

Comment:
This is more a FYI. For a lack of a better name, we are thinking adding a semantics action that call DefaultAction to support ios magic tap. https://developer.apple.com/documentation/objectivec/nsobject/1615137-accessibilityperformmagictap?language=objc.

I am not sure how should they overlap here. Is the Action::DEFAULT always referring to tap in fuchsia?

Reply:
Thanks for the heads-up. Based on the description in that link, it looks like Action::Default is the most appropriate mapping for SemanticsAction::kTap.